packages/components/src/SvgIcon/index.tsx

```diff
@@ -1 +1 @@
-export { default } from "./SvgIcon";
+export { default, IconProps } from "./SvgIcon";
```

Comment:
does this work fine now, or should we still export Props on a separate line?

Reply:
Good call, I forgot there was an issue. Everything is green, and I don't see TypeScript or Babel complaining about anything.

But when I look at the dist directory, there is an export for `_SvgIcon.IconProps`. I'll move this to its own line to prevent that from showing up in the js file at all.